Vidyano.Core/ViewModel/PersistentObjectAttribute.cs

```diff
@@ -206,7 +211,12 @@ internal AttributeVisibility Visibility
         internal bool UpdateValue(object value)
         {
             if (IsReadOnly)
+            {
+                if (Client.StrictMode)
```

Comment:
same logic will apply here, we _don't_ want to throw if the value was the same anyway